src/main/groovy/com/ableton/JenkinsMocks.groovy

```diff
@@ -80,8 +80,8 @@ class JenkinsMocks {
       return output.stdout
     } else if (returnStatus) {
       return output.exitValue
-    } else {
-      return output.exitValue == 0
```

Comment:
Please change the above `else if` into a simple `if` as well.

Reply:
@ala-ableton Done!

Reply:
I was expecting a single commits with both changes (with a `Do not use else after return` kind of message), but the current commits are fine as well.